src/main/kotlin/org/malachite/estella/people/api/JobSeekerController.kt

```diff
@@ -0,0 +1,73 @@
+package org.malachite.estella.people.api
+
+import org.malachite.estella.commons.EStellaHeaders
+import org.malachite.estella.commons.Message
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.commons.SuccessMessage
+import org.malachite.estella.commons.models.people.JobSeeker
+import org.malachite.estella.commons.models.people.User
+import org.malachite.estella.services.JobSeekerService
+import org.malachite.estella.services.SecurityService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/jobseekers")
+class JobSeekerController(
+    @Autowired private val jobSeekerService: JobSeekerService,
+    @Autowired private val securityService: SecurityService
+) {
+
+    @CrossOrigin
+    @GetMapping
+    fun getJobSeekers(): ResponseEntity<MutableIterable<JobSeeker>> =
+        jobSeekerService.getJobSeekers().let {
+            ResponseEntity(it, HttpStatus.OK)
+        }
+
+    @CrossOrigin
+    @GetMapping("/{jobseekerId}")
+    fun getJobSeekerById(@PathVariable("jobseekerId") jobSeekerId: Int): ResponseEntity<JobSeeker> =
+        jobSeekerService.getJobSeeker(jobSeekerId).let {
+            ResponseEntity(it, HttpStatus.OK)
+        }
+
+    @CrossOrigin
+    @PostMapping()
```

Comment:
Ogólnie nie wiem czy nie trzeba by się trzymać jakiejś konwencji jeśli chodzi o nazwy endpointów. Bo czasami jest add coś, a czasami nie, tak samo z update

Reply:
To musimy przegadać jeszcze, to na razie tego nie poprawiam, bo nie wiem na co. Ogólnie poruszmy ten temat w poniedziałek, albo we wtorek.